docs/status.md

```diff
@@ -17,16 +17,26 @@ particular aspect, as mentioned in the corresponding column, is tracked using
 one of the following tracking labels.
 
  - Generic labels
-    - **yes**: complete
-    - **no**: not complete yet, but part of [the roadmap](https://github.com/openxla/stablehlo#roadmap).
+    - **yes**: there is a comprehensive implementation.
+    - **no**: there is no implementation, but working on that is part of
+      [the roadmap](https://github.com/openxla/stablehlo#roadmap).
+      Note that Verifier can never be labeled as "no" because the ODS already
+      implements some verification.
  - Customized labels for Verifier and Type Inference
-    - **yes**: in sync with [StableHLO semantics](https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md).
-    - **yes\***: in sync with [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
-    - **revisit**: there is an implementation in between
-      [the ODS](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.td)
-      and [StablehloOps.cpp](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.cpp)
-      but it needs to be revisited to determine its status.
-    - **infeasible**: infeasible to implement by design.
+    - **yes**: there is an implementation, and it's in sync with
+      [StableHLO semantics](https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md).
+    - **yes\***: there is an implementation, and it's in sync with
+      [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
+      Since XLA semantics is oftentimes underdocumented, we are using
+      [hlo_verifier.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/hlo_verifier.cc)
+      and [shape_inference.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/shape_inference.cc)
+      as the reference.
+    - **revisit**: there is an implementation, but it doesn't fall under "yes"
+      or "yes\*" - either because we haven't audited it yet, or because we have
+      and found issues.
```

Comment:
One question to clarify: about ` we have and found issues.`

As we already found issues, there must be a ticket for it. Now we might have two options 

"yes*": At this point the semantics is same as HLO.
"revisit": There might be other issue and need need deeper investigation.

Reply:
"Now we might have two options: "yes*": At this point the semantics is same as HLO...". That cannot be the case by definition because in the earlier part of the sentence we say "there is an implementation, but it doesn't fall under "yes" or "yes\*"".